tempodb/backend/local/local.go

```diff
@@ -161,9 +161,27 @@ func (rw *Backend) ReadRange(ctx context.Context, name string, keypath backend.K
 	return nil
 }
 
-// Shutdown implements backend.Reader
+// Shutdown implements backend.Reader. It attempts to clear all tenants
+// that do not have blocks.
 func (rw *Backend) Shutdown() {
```

Comment:
Forgot this existed.  Excellent.

Reply:
Yeah. It's weird that `Shutdown()` on the Reader interface does the job of cleaning up, but meh. We should combine these interfaces anyway. I don't think much is gained from the split up.

(or just add Shutdown to the writer?)